src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  const minimumDiscount = 20;
+  const maximumDiscount = 50;
```

Comment:
Use constants instead of literals for the number of days that determine the discount thresholds. This will make the code more maintainable and self-explanatory.